### PR TITLE
Python: fix undefined name by pyflakes found

### PR DESCRIPTION
**Pull request by thinkerou**

### CODE REVIEW

**Comment by soltanmm** [comment]
Thanks for the find! Though, the preferred fix would be to `from distutils import errors` at the top and to replace the use of `DistutilsOptionError` with `errors.DistutilsOptionError`.

**Comment by thinkerou** [comment]
Thanks @soltanmm , I have recommited.

File: src/python/grpcio_tests/commands.py [hunk 1]
```diff
@@ -41,6 +40,7 @@
 import traceback
 
 import setuptools
+from distutils import errors
```
Comment (soltanmm) [inline src/python/grpcio_tests/commands.py]: `distutils` is a standard Python library, so this import should be right back up before `glob`. Let's try not to run away from the style guide too much when it's easy not to.

Comment (thinkerou) [inline src/python/grpcio_tests/commands.py]: Fixed, thanks!

**Comment by nathanielmanistaatgoogle** [comment]
Please squash commits and please follow [these guidelines](http://chris.beams.io/posts/git-commit/#seven-rules) in your commit message.

**Comment by nathanielmanistaatgoogle** [comment]
There should just be one commit in [the commits tab](https://github.com/grpc/grpc/pull/7432/commits) of this pull request - at the moment there are two?

**Comment by nathanielmanistaatgoogle** [comment]
Wait, why close the pull request?

**Comment by thinkerou** [comment]
@nathanielmanistaatgoogle Sorry, I squash commits and I am not very skillful.
It's OK now.

**Comment by thinkerou** [comment]
@soltanmm Please review the PR right? Thanks!

**Comment by soltanmm** [comment]
Sorry about the long turnaround. Please squash commits?

**Comment by thinkerou** [comment]
OK, done, thanks!

**Comment by thinkerou** [comment]
@soltanmm I have squashed  the PR, please review, thanks.

**Comment by nathanielmanistaatgoogle** [comment]
Apologies for the long turnaround (again) on this. Please resolve conflicts?

**Comment by thinkerou** [comment]
@nathanielmanistaatgoogle fixed.

**Comment by nathanielmanistaatgoogle** [comment]
... and squash commits, please. :smile:

**Comment by thinkerou** [comment]
squash done.

**Comment by nathanielmanistaatgoogle** [comment]
Premerge test assessment:

* [`grpc_performance_pull_requests` is in very poor shape](https://github.com/grpc/grpc/issues/9638)

**Comment by nathanielmanistaatgoogle** [comment]
Thanks much!

**Pull request merged**